tools/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -223,7 +224,13 @@ export class FluidPackageCheck {
 
             const buildPrefix = pkg.getScript("build:genver") ? "npm run build:genver && " : "";
             if (pkg.getScript("tsc")) {
-                buildCompile.push("tsc");
+                if (pkg.getScript("build:test")) {
+                    buildCommonJs.push("tsc");
+                    buildCommonJs.push("build:test");
+                    buildCompile.push("build:commonjs");
```

Comment:
Curious, why is this needed?

Reply:
This is to prepare for having a new script in the package.json `build:test` for building the tests separate from the source.  This is here to enforce some uniformity of the packages if they have the build:test script.